src/assets/sass/_header.scss

```diff
@@ -0,0 +1,109 @@
+@media only screen and (min-width: 769px){
+    .header-container {
+        display: block;
+    }
+}
+.header-container .header-main {
+    color: #0b0c0c;
```

Comment:
I believe that we can remove this, isn't it?

Reply:
As per figma design this is the text color.

<img width="742" alt="image" src="https://github.com/DEFRA/ncea-frontend/assets/149910290/f943a4ff-65a7-47a2-a228-622ee97bf927">